chipy_org/apps/meetings/admin.py

```diff
@@ -79,7 +79,7 @@ class Media:
 
 class PresentorAdmin(admin.ModelAdmin):
     list_display = [
-        'id', 'user', 'email', 'phone']
+        'id', 'name', 'user', 'email', 'phone']
```

Comment:
Adding name field to the Presenter List, as it's possible that this is the only field filled out by a presenter and the list page would contain an empty looking record otherwise.